extension-push/src/java/com/defold/push/Push.java

```diff
@@ -157,16 +157,18 @@ public void stop() {
     }
 
     public void flushStoredNotifications() {
-        for (int i = 0; i < storedNotifications.size(); i++) {
-            StoredNotification n = storedNotifications.get(i);
-            if (n.wasLocal) {
-                this.listener.onLocalMessage(n.json, n.id, n.wasActivated);
-            } else {
-                this.listener.onMessage(n.json, n.wasActivated);
+        if (this.listener != null) {
```

Comment:
Tiny nitpick for the future.
We prefer early-outs, since I think they are easier to read and reason about.

E.g. 
```
if ( ! thing ) return;

stuff
```